fuzzbench/worker.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Self-defined worker module."""
+import time
+
+import redis
+import rq
+
+REASSIGN_GAP_TIME = 5
+
+
+def main():
+    """Sets up Redis connection and starts the worker."""
+    redis_connection = redis.Redis(host="queue-server")
+    with rq.Connection(redis_connection):
+        queue = rq.Queue('build_n_run_queue')
+        worker = rq.Worker([queue], connection=redis_connection)
+
+        while queue.count + queue.deferred_job_registry.count > 0:
+            worker.work(burst=True)
+            time.sleep(REASSIGN_GAP_TIME)
```

Comment:
i think `REASSIGN_GAP_TIME` just raises more questions for the reader than answers, i'd probably just `time.sleep(5)` here. i think we want to sleep here a bit no matter what, otherwise this (potentially empty) loop  might eat up the CPU

Reply:
OK. Done.